index.bs

```diff
@@ -1196,7 +1209,7 @@ controls the maximum [=map/size=] of an [=attribution source=]'s
 
 <dfn>Max length per aggregation key identifier</dfn> is a positive integer that controls
 the maximum [=string/length=] of an [=attribution source=]'s [=attribution source/aggregation keys=]'s
-[=map/keys=], an [=attribution trigger=]'s [=attribution trigger/aggregatable values=]'s [=map/keys=],
+[=map/keys=], an [=attribution trigger=]'s [=attribution trigger/aggregatable values configurations=]'s [=map/keys=],
```

Comment:
I think this is not right. It should be the keys of configurations's items's values field.

Reply:
Nice catch!